MATLAB/orbit_estimation/+Orbit/abortpropagator.m

```diff
@@ -0,0 +1,7 @@
+function [self] = abortpropagator(self)
+%ABORTPROPAGATOR Aborts the propagator,
+%   If self is propagating this restores self to before
+%   startpropagator was called. If self is not propagating, this does
+%   nothing.
+end
```

Comment:
I'm assuming this is used if you want to unapply a deltav or something like that?